### ISSUE TITLE
How I can config a virtual host inside the slingshot?

### ISSUE THREAD

**Issue by jmoguelruiz** [coryhouse/react-slingshot#423]
I need generate a virtual host like project.company.dev when run the command npm start, actually appear localhost::3000 and when I try to get resource of amazon s3 like a font, display the follow error:

'https://d224....cloudfront.net/themes/default/css/fonts/font_one.woff' from origin 'http://localhost:3000' has been blocked by CORS policy: The 'Access-Control-Allow-Origin' header has a value 'https://coupon.company.com' that is not equal to the supplied origin. Origin 'http://localhost:3000' is therefore not allowed access.

I dont want add localhost in the s3, my company follows standars.

**Comment by nickytonline** [comment]
@jmoguelruiz, this is out of the scope of the project but here some stuff to get you going on the path you want. You can modify your hosts file (works on Windows, Linux and Mac) and add an entry for your domain.

```
127.0.0.1      project.company.dev
```
and remove the port the config, https://github.com/coryhouse/react-slingshot/blob/master/tools/srcServer.js#L18 . It should default to port 80.

Hope this gets you started in the right direction.

**Comment by nickytonline** [comment]
Closed as out of scope of the project.

**Comment by jmoguelruiz** [comment]
Thank you very much @nickytonline  for your help, I ended up configuring my virtual host by adding the following configuration in httpd-vhosts.conf.
```php 
<VirtualHost localhost:80>
    ServerName project.company
    ServerAlias project.company.dev

    ProxyRequests off

    <Proxy *>
        Order deny,allow
        Allow from all
    </Proxy>

    <Location />
        ProxyPass http://localhost:3000/
        ProxyPassReverse http://localhost:3000/
    </Location>
</VirtualHost>
```
So I did not have to modify the file srcServer.js

**Comment by nickytonline** [comment]
Cool stuff! Gald you're up and running @jmoguelruiz.
